layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationNewPageElementItemsList.vue

```diff
@@ -59,6 +65,7 @@ export default {
       templates: null,
       expanded: false,
       collator: new Intl.Collator(eXo.env.portal.language, {numeric: true, sensitivity: 'base'}),
+      selectedTemplateId: null
```

Comment:
should add a `watch` on this to be able to `$emit(''input)` on it then use it on parent component, else the selected template will not be recognized in parent compoennt.

Reply:
done